tests/ert_tests/conftest.py

```diff
@@ -1 +1,15 @@
 # This conftest still exists so that tests files can import ert_utils
+import logging
+import pytest
+
+
+@pytest.fixture(autouse=True)
+def log_check():
+    logger = logging.getLogger()
+    logger.setLevel(logging.WARNING)
```

Comment:
Isnt this a bit high? Should be set to `DEBUG` to catch if any tests are actually modifying the log level?

Reply:
Warning is the default in pytest it seems, so leaving it at that